### ISSUE TITLE
The loss during training is nan. 

### ISSUE THREAD

**Issue by tingyangsh** [jwchoi384/Gaussian_YOLOv3#62]
Thank you for your contribution. I have a question to ask. When I am training, loss and avg loss appear nan, do you have any suggestions?The parameter I set: learning rate = 0.0001, width = 608 height = 608.the problem is as follows:
1031: 32.407806, 9997.750000 avg, 0.000100 rate, 2.301689 seconds, 32992 images
1032: 50.673481, 9003.041992 avg, 0.000100 rate, 2.374657 seconds, 33024 images
1033: -nan, -nan avg, 0.000100 rate, 2.386819 seconds, 33056 images
1034: -nan, -nan avg, 0.000100 rate, 2.371522 seconds, 33088 images
1035: nan, -nan avg, 0.000100 rate, 2.369323 seconds, 33120 images
1036: nan, -nan avg, 0.000100 rate, 2.363159 seconds, 33152 images
1037: nan, -nan avg, 0.000100 rate, 2.366278 seconds, 33184 images
1038: nan, -nan avg, 0.000100 rate, 2.365389 seconds, 33216 images
1039: nan, -nan avg, 0.000100 rate, 2.368548 seconds, 33248 images
1040: nan, -nan avg, 0.000100 rate, 2.361531 seconds, 33280 images
Resizing

**Comment by jwchoi384** [comment]
@tingyangsh 
Hi, 
Probably because of the variance.
Please see the [link](https://github.com/jwchoi384/Gaussian_YOLOv3/issues/16#issuecomment-549228854).
